vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -71,6 +71,7 @@ def build(self):
                     value=self.value if self.value is not None else default_value,
                     multi=self.multi,
                     persistence=True,
+                    optionHeight=32 if all(len(option) <= 37 for option in full_options) else 60,  # noqa: PLR2004
```

Comment:
Where do these numbers come from?

Fine if 37 is just a rough heuristic about what to treat as a cutoff, but just curious where it comes from. I guess 32 is what we have now, but then why 60 for a double height line rather than 64?

Presumably if someone does a really long option this will suffer from the same problem we currently have - is there anything more general that would handle that too? e.g. we could do the sort of analytic continuity of your step function like this if you see what I mean: https://www.desmos.com/calculator/c5fh81syid. People are unlikely to (or shouldn't be encouraged to) do huge long values, but if it's easy to accommodate a more general case rather than just a single "critical value" of 37 then we might as well.

Reply:
Ah yeah, I like that idea! I'll implement that.

To your question - I'll add some comments there as well. 37 is the cut off value where the text starts to wrap on the next line. This is determined by our left-side panel width and dropdown width. If this changes, that number also has to change. I could only determine it manually though.

32 and 60 might still change - I have to take a look at the screenshots later. The reason why it's not 32 and then 64 is padding. The font-size remains the same, so the top and bottom padding should be the same as in the case of only 1-line of text. If you would choose 64, the padding-top and bottom would change. So this is more a design guidance, but I am not super strict about it.

Reply:
In the end, it was 32 and 56 :)